### PR TITLE
Spread Option pricing

### PR DESCRIPTION
**Pull request by abhinav1630**

### CODE REVIEW

**Review by jkirkby3 — COMMENTED** [review]
Please add type hinting to the function signature and response, otw looks great!

**Comment by abhinav1630** [comment]
Updated the type hinting. Will add the tests

**Review by abhinav1630 — COMMENTED** [review]
Updated the type hinting

File: fypy/pricing/analytical/spread_option_kirk_method.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                              T : float,
+                              r: float,
+                              rho: float,
+                              sigma_1: float,
+                              sigma_2: float,
+                              q_1: float,
+                              q_2: float) -> float:
+    """
+    Params:
+    K: Option Strike
```
Comment (nrupprecht) [inline fypy/pricing/analytical/spread_option_kirk_method.py]: @jkirkby3  do we expect param commenting like
_:param K: Option strike
:param S0_1: Initial Price of asset 1_
etc.?

Comment (jkirkby3) [reply fypy/pricing/analytical/spread_option_kirk_method.py]: Yea nice catch @nrupprecht ... @abhinav1630 , can you update the documentation style to match the rest of library? You can configure this in your IDE.

**Review by abhinav1630 — COMMENTED** [review]
Updated the param description

File: fypy/pricing/analytical/spread_option_kirk_method.py (hunk 1 above)
Comment (abhinav1630) [reply fypy/pricing/analytical/spread_option_kirk_method.py]: Thanks @nrupprecht. Have updated that

**Review by jkirkby3 — APPROVED** [review]

**Pull request merged**